packages/plugin-ebarimt-api/src/models/ProductRule.ts

```
@@ -0,0 +1,48 @@
+        { _id },
+        {
+          $set: {
+            ...doc,
+            modifiedAt: new Date()
+          }
+        }
+      );
+
+      return response;
+    }
+  }
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)